subsys/bluetooth/controller/Kconfig

```diff
@@ -1008,13 +1008,13 @@ config BT_CTLR_CONN_ISO_PDU_LEN_MAX
 	help
 	  Maximum Connected Isochronous Channel PDU Length.
 
-config BT_CTLR_CONN_ISO_SDU_LEN_MAX
-	int "Maximum Connected Isochronous Channel SDU Length"
-	depends on BT_CTLR_CONN_ISO
+config BT_CTLR_ISO_SDU_LEN_MAX
+	int "Maximum Isochronous Channel SDU Length"
+	depends on BT_CTLR_BROADCAST_ISO || BT_CTLR_CONN_ISO
```

Comment:
Is this just used for TX? If so, then perhaps `BT_CTLR_BROADCAST_ISO` should be `CONFIG_BT_CTLR_ADV_ISO`?

Reply:
True, fixed and renamed to `BT_CTLR_ISO_TX_SDU_LEN_MAX`